dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -914,6 +914,55 @@ def run(*_args, &_block)
     assert_logged 'Cleared 2 CodeReviewComment'
   end
 
+  #
+  # Table: dashboard.code_review_requests
+  # Table: dashboard.code_review_notes
+  #
+  test "deletes comment text and soft deletes comments for purged user" do
+    student = create :student
+    review = create :code_review, user_id: student.id
+    student_2 = create :student
+    comment = create :code_review_note, commenter: student_2, code_review: review
+    assert_nil review.deleted_at
+    assert_nil comment.deleted_at
+    refute_nil comment.comment
+
+    purge_user student
+    review.reload
+    comment.reload
+    # assert that the review and comment were soft-deleted, and the comment text was hard-deleted.
+    refute_nil review.deleted_at
+    refute_nil comment.deleted_at
+    assert_nil comment.comment
+    # the commenter was not deleted, so their id should remain on the comment
+    refute_nil comment.commenter
+  end
+
+  test "anonymizes and deletes code review comments written by user" do
```

Comment:
I guess we don't need to anonymize them anymore since they're just getting deleted

Reply:
since we are soft-deleting I'm still anonymizing